.env.example

```diff
@@ -1,2 +1,4 @@
 OPENAI_API_KEY=your_openai_api_key_here
 N_IMAGES=5
+MIN_IMAGE_WIDTH=100
```

Comment:
Make this 300x250

Reply:
Sure, I will update the minimum image dimensions to 300x250 as requested. Thank you for the feedback!